db/helpers/collectorUtils.js

```diff
@@ -82,8 +82,26 @@ function validateVersion(version) {
   }
 }
 
+/**
+ * Find unassigned generators and assign them.
+ * @param {Object} seq - Sequelize object
+ * @returns {Promise} - Resolves to array of assigned generator db objects
+ */
+function findAndAssignGenerators(seq) {
```

Comment:
It's not clear from the function name that this only applies to unassigned generators.